src/track/cue.cpp

```diff
@@ -88,7 +88,8 @@ Cue::Cue(
 Cue::Cue(
         const mixxx::CueInfo& cueInfo,
         mixxx::audio::SampleRate sampleRate)
-        : m_bDirty(false),
+        // Mark imported cues dirty before storing them in the database
```

Comment:
The comment is hard to understand without the calling context, which is out of sight here.
Can this class decide if the cue is dirty or not? 
I think it would be better if the caller sets it as a Parameter.

Reply:
I agree. The caller should explicitly decide if the cue object is considered dirty or not.

Reply:
I think the comment is now outdated.